### PR TITLE
Netty version bump

### LINKED ISSUE #281: Bump netty version 4.1.78

### PR DESCRIPTION
**Pull request by Elishtar**
I haven't forgotten about:
- [ ] Tests
- [x] Changelog
- [ ] Documentation
- [ ] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:

Closes #281

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+- ### Misc
```
Comment (ArtDu) [inline CHANGELOG.md:3]: Move it below, above `## [Unreleased]`

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]
See comment from @ArtDu

**Review by dkasimovskiy — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Elishtar) [reply CHANGELOG.md]: Moved

**Comment by Elishtar** [comment]
> See comment from @ArtDu

Fixed

**Pull request merged**